Sources/Sentry/SentryNetworkSwizzling.m

```diff
@@ -8,6 +9,7 @@ + (void)start
 {
     [SentryNetworkTracker.sharedInstance enable];
     [self swizzleURLSessionTaskResume];
+    [self swizzleURLSessionWithConfiguration];
```

Comment:
`m`: Isn't registering in `NSURLProtocol` enough? Why do we need to swizzle here?

Reply:
NSURLProtocol is only used when the dev uses NSURLSession.shared.
If the dev creates a custom NSURLSession with custom configurations or protocol is not called.

Reply:
Again please add this info as a comment please 😁